Packs/Gmail/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### Gmail
+- **Note**: The argument *show-only-mailboxes* will now be True by default when using the command ***gmail-search-all-mailboxes***.
+- Added support for searching large numbers of mailboxes in ***gmail-search-all-mailboxes***, please refer to the command's description for more information.
+#### Playbooks
+##### New: Search all mailboxes - Gmail with polling
+- Added the new playbook. This playbook searches Gmail records for all Google users, designed for large companies with over 2500 Google users. (Available from Cortex XSOAR 6.5.0).
+##### New: Search in mailboxes Gmail (Loop) with polling
+- Added the new playbook. This playbook is used as a subplaybook for the Playbook Search-all-mailboxes - Gmail, it should not be used. (Available from Cortex XSOAR 6.5.0).
```

Comment:
@israelpoli - what do you mean by "it should not be used" - what should not be used?

Reply:
@julieschwartz18 

It's a subplaybook that when you run it alone it crashes,
Run this subplaybook only through the parent playbook.